From 12a9288c72c5f2ac57192b67fea714978f45a140 Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Tue, 30 Jan 2024 09:35:23 +0100
Subject: [PATCH] Add CSS patch to drop pointer-events dfn from CSS UI

The definition of `pointer-events` is not yet as complete as that in the SVG
spec. Tracking issue at:
https://github.com/w3c/csswg-drafts/issues/4438
---
 ed/css/css-ui.json | 32 --------------------------------
 1 file changed, 32 deletions(-)

diff --git a/ed/css/css-ui.json b/ed/css/css-ui.json
index 0532a7609..4dc9009de 100644
--- a/ed/css/css-ui.json
+++ b/ed/css/css-ui.json
@@ -562,38 +562,6 @@
         "userSelect"
       ]
     },
-    {
-      "name": "pointer-events",
-      "href": "https://drafts.csswg.org/css-ui-4/#propdef-pointer-events",
-      "value": "auto | none",
-      "initial": "auto",
-      "appliesTo": "all elements",
-      "inherited": "yes",
-      "percentages": "N/A",
-      "computedValue": "specified keyword",
-      "canonicalOrder": "per grammar",
-      "animationType": "by computed value type",
-      "values": [
-        {
-          "name": "auto",
-          "prose": "Hit-testing occurs normally.",
-          "href": "https://drafts.csswg.org/css-ui-4/#valdef-pointer-events-auto",
-          "type": "value",
-          "value": "auto"
-        },
-        {
-          "name": "none",
-          "prose": "Hit-testing acts as if the boxes generated by the element (see [CSS-DISPLAY-3]) were not there, effectively causing the element behind the pointer-events: none element to become the target of hit-testing based events instead. The statement above needs to be be qualified; for some purposes, hit-testing continues to take the element into account. For instance, clicking and dragging to start a text selection will continue to work as usual. What is the precise set of things for which hit-testing is affected by pointer-events?",
-          "href": "https://drafts.csswg.org/css-ui-4/#valdef-pointer-events-none",
-          "type": "value",
-          "value": "none"
-        }
-      ],
-      "styleDeclaration": [
-        "pointer-events",
-        "pointerEvents"
-      ]
-    },
     {
       "name": "accent-color",
       "href": "https://drafts.csswg.org/css-ui-4/#propdef-accent-color",
-- 
2.42.0.windows.2

